include/mbedtls/config.h

```diff
@@ -1784,6 +1784,14 @@
  */
 #define MBEDTLS_CTR_DRBG_C
 
+/**
+ * \def MBEDTLS_CTR_DRBG_ABI_COMPAT
+ *
+ * Enables compatible ABI for DRBG. 
+ * Disabling this shrinks the size of mbedtls_ctr_drbg_context by ~248 bytes.
+ */
+#define MBEDTLS_CTR_DRBG_ABI_COMPAT
```

Comment:
Please generate the `version_features.c` file and push it as well:
~~~
./scripts/generate_features.pl 
~~~
This should resolve your CI failure